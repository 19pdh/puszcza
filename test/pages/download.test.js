import { shallowMount } from '@vue/test-utils'
import download from '../../pages/download'

describe('download', () => {
  // Now mount the component and you have the wrapper
  const wrapper = shallowMount(download)

  it('match snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
