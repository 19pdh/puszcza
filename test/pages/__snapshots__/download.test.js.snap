// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`download match snapshot 1`] = `
<div
  class="container"
>
  <h1>
    Do pobrania
  </h1>
   
  <a
    class="link"
    href="/files/Karta_próby_na_sprawność.pdf"
  >
    Karta próby na sprawność
  </a>
   
  <a
    class="link"
    href="/files/Próba_Harcerza.pdf"
  >
    Próba Harcerza
  </a>
</div>
`;
